apps/parsley/src/components/LogWindow/index.tsx

```diff
@@ -6,6 +7,14 @@ import { ParsleyRow } from "components/LogRow/RowRenderer";
 import SidePanel from "components/SidePanel";
 import SubHeader from "components/SubHeader";
 import { useLogContext } from "context/LogContext";
+import { useSectionsFeatureDiscoveryContext } from "context/SectionsFeatureDiscoveryContext";
+
+const SectionsFeatureModal = lazy(
+  () =>
+    import(
+      "../../context/SectionsFeatureDiscoveryContext/SectionsFeatureModal"
```

Comment:
I saw the Loadable component and didn't think it made sense for this implementation. Since the modal is an overlay onto page content, it doesn't make sense design-wise to apply the full page load fallback that is built into Loadable.

Reply:
Nice, I added in the absolute import. For some reason my ts-server wasn't understanding it earlier.